### PR TITLE
test: trade factory position handler

### PR DESCRIPTION
**Pull request by b0dhidharma**

### CODE REVIEW

**Review by lbertenasco — CHANGES_REQUESTED** [review]
Why did you changed _owner to _strategy?
because the "user" is a year's strategy?
I'd personally rather have it as user or owner, since _strategy kinda confuses me with swapping strategy.
wdyt?

**Comment by b0dhidharma** [comment]
> Why did you changed _owner to _strategy?
> because the "user" is a year's strategy?
> I'd personally rather have it as user or owner, since _strategy kinda confuses me with swapping strategy.
> wdyt?

Yeah, actually I thought `_owner` was kinda misleading ... Since we have `onlyStrategy` and stuff ... It's like ... "There is not gonna be some owner thats not going to be a strategy, at least for now".
But I don't have strong feelings about it, so, I can change it if you do have them !

File: contracts/TradeFactory/TradeFactoryPositionsHandler.sol [hunk 1]
```diff
@@ -95,51 +96,98 @@ abstract contract TradeFactoryPositionsHandler is ITradeFactoryPositionsHandler
...
+      pendingTradesById[_pendingTradesByOwner[msg.sender].at(i)]._swapper = _swapperAddress;
+      _changedSwapperIds[i] = _pendingTradesByOwner[msg.sender].at(i);
+    }
+    emit TradesSwapperChanged(msg.sender, _changedSwapperIds, _swapper);
+  }
+
+  function setSwapperSafetyCheckpoint(uint256 _checkpoint) external override onlyStrategy {
     require(_checkpoint <= block.timestamp, 'TradeFactory: invalid checkpoint');
-    swapperSafetyCheckpoint[_to] = _checkpoint;
+    swapperSafetyCheckpoint[msg.sender] = _checkpoint;
```
Comment (lbertenasco) [inline contracts/TradeFactory/TradeFactoryPositionsHandler.sol:163]: nice catch

**Comment by lbertenasco** [comment]
> > Why did you changed _owner to _strategy?
> > because the "user" is a year's strategy?
> > I'd personally rather have it as user or owner, since _strategy kinda confuses me with swapping strategy.
> > wdyt?
> 
> Yeah, actually I thought `_owner` was kinda misleading ... Since we have `onlyStrategy` and stuff ... It's like ... "There is not gonna be some owner thats not going to be a strategy, at least for now".
> But I don't have strong feelings about it, so, I can change it if you do have them !

yeah, for us might be easier to differentiate. ok let's keep strategy.

**Review by lbertenasco — APPROVED** [review]

**Pull request merged**